Add a Pixie Plugin section to the Troubleshooting guide. (#182)
pixie-labs/pixie-docs: content/en/04-tutorials/03-integrations/03-otel.md

```

Congrats! You're now exporting Pixie data to an OTel collector.

## Troubleshooting

> Please check out the **Pixie Plugin** section of our [Troubleshooting](/about-pixie/troubleshooting/#troubleshooting-troubleshooting-a-pixie-plugin) guide.

Reviewer: ```suggestion
Having problems?  Check out the [Pixie Plugin Troubleshooting](/about-pixie/troubleshooting/#troubleshooting-troubleshooting-a-pixie-plugin) guide.
```
